jujugui/static/gui/src/app/components/error-boundary/test-error-boundary.js

```diff
@@ -0,0 +1,47 @@
+/* Copyright (C) 2018 Canonical Ltd. */
+'use strict';
+
+const React = require('react');
+const enzyme = require('enzyme');
+const ErrorBoundary = require('./error-boundary');
+
+class ComponentWithError extends React.Component {
+  constructor(props) {
+    super(props);
+    throw new Error('Error!');
+  }
+  render() {
+    return (
+      <div>
+        <h1>Buggy component</h1>
+      </div>
+    );
+  }
+}
+
+describe('Error boundary', function() {
+
+  const renderComponent = (options = {}) => enzyme.mount(
+    <ErrorBoundary>
+      {options.content || null}
+    </ErrorBoundary>
+  );
+
+  it('should not show if no error thrown', () => {
+    const wrapper = renderComponent({
+      content: (<span>Children content!</span>)
+    });
+    expect(wrapper).toMatchSnapshot();
+  });
+
+  it('should show fallback UI if error thrown', () => {
+    const wrapper = renderComponent({
+      content: (<ComponentWithError />)
+    });
+    expect(wrapper).toMatchSnapshot();
+  });
+
+  afterEach(() => {
```

Comment:
At a risk of being a little pedantic, would you mind moving this up above any of the `it()` tests. It's kind of a convention, then you can easily see the test setup and teardowns before you get to the tests.